src/control/server/faults.go

```diff
@@ -73,6 +73,28 @@ func FaultScmUnmanaged(mntPoint string) *fault.Fault {
 	)
 }
 
+func FaultBdevNotFound(bdevs []string) *fault.Fault {
+	plural := ""
+	if len(bdevs) > 1 {
+		plural = "s"
+	}
```

Comment:
This is a nice touch. Worth noting that go-humanize includes an "english" sub-package with handy utility functions for this kind of thing: https://godoc.org/github.com/dustin/go-humanize/english